### PR TITLE
Added Doc for UserStoppedTypingBehavior

### PR DESCRIPTION
**Pull request by CliffAgius**
New doc with Index and TOC for the UserStoppedTypingBehavior for open proposal:

    https://github.com/CommunityToolkit/Maui/issues/49

### CODE REVIEW

**Comment by PRMerger-2** [comment]
## PRMerger Results
 | Issue | Description |
 --- | --- |
 Added File(s) | This PR contains added files. New files require human review.
 Yaml File(s) | This PR includes changes to .yml file(s) owned by another author.
 File Change Percent | This PR contains file(s) with more than 20% file change.
 Index File | This PR contains a change to the index file.

File: docs/maui/behaviors/user-stopped-typing-behavior.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+author: cliffagius
+description: "The UserStoppedTypingBehavior is a Behavior that allows the user to trigger an action when a user has stopped data input an Entry. Examples of its usage include triggering a search when a user has stopped entering their search query."
+ms.date: 04/13/2022
+---
+
+# UserStoppedTypingBehavior
+
+[!INCLUDE [docs under construction](../includes/preview-note.md)]
+
+The `UserStoppedTypingBehavior` is a `Behavior` that allows the user to trigger an action when a user has stopped data input an `Entry`. Examples of its usage include triggering a search when a user has stopped entering their search query.
```
Comment (jfversluis) [inline docs/maui/behaviors/user-stopped-typing-behavior.md:12]: Should we make this a bit more generic? Or at least point out that `Entry` is only one of the examples? Users might wrongfully think that this only works with an `Entry` while it also does with `SearchBar`, `Editor`, etc.

Additionally, I think the technical term is "debounce" for this. Should we mention that somewhere?

File: docs/maui/behaviors/user-stopped-typing-behavior.md [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+{
+    public UserStoppedTypingBehaviorPage()
+    {
+        var behavior = new UserStoppedTypingBehavior()
+        {
+            StoppedTypingTimeThreshold = 1000,
+            MinimumLengthThreshold = 3,
+            ShouldDismissKeyboardAutomatically = true
+        };
+        behavior.SetBinding(UserStoppedTypingBehavior.CommandProperty, nameof(ViewModel. SearchCommand);
```
Comment (jfversluis) [inline docs/maui/behaviors/user-stopped-typing-behavior.md:53]:
```suggestion
        behavior.SetBinding(UserStoppedTypingBehavior.CommandProperty, nameof(ViewModel.SearchCommand);
```

File: docs/maui/behaviors/user-stopped-typing-behavior.md [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+            Placeholder = "Start typing when you stop the behavior will trigger..."
+        };
+        entry.Behaviors.Add(behavior);
+    }
+}
+```
+
+### C# Markup
+
+Our [`CommunityToolkit.Maui.Markup`](../markup/markup.md) package provides a much more concise way to use this converter in C#.
```
Comment (jfversluis) [inline docs/maui/behaviors/user-stopped-typing-behavior.md:66]:
```suggestion
Our [`CommunityToolkit.Maui.Markup`](../markup/markup.md) package provides a much more concise way to use this behavior in C#.
```

File: docs/maui/behaviors/user-stopped-typing-behavior.md [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+                Placeholder = "Start typing when you stop the behavior will trigger..."
+            }
+            .Behaviors(new UserStoppedTypingBehavior
+            {
+                StoppedTypingTimeThreshold = 1000,
+                MinimumLengthThreshold = 3,
+                ShouldDismissKeyboardAutomatically = true
+            }.Bind(
+                UserStoppedTypingBehavior.CommandProperty, 
+                nameof(ViewModel. SearchCommand));
```
Comment (jfversluis) [inline docs/maui/behaviors/user-stopped-typing-behavior.md:86]:
```suggestion
        Content = new Entry
        {
            Placeholder = "Start typing when you stop the behavior will trigger..."
        }
        .Behaviors(new UserStoppedTypingBehavior
        {
            StoppedTypingTimeThreshold = 1000,
            MinimumLengthThreshold = 3,
            ShouldDismissKeyboardAutomatically = true
        }
        .Bind(
                UserStoppedTypingBehavior.CommandProperty, 
                nameof(ViewModel. SearchCommand));                   
```

Minor code formatting fixes. Not sure if this is better, but at least more consistent? 😅

**Review by jfversluis — CHANGES_REQUESTED** [review]
Some minor things, should be good after that, thank you once again!

File: docs/maui/behaviors/user-stopped-typing-behavior.md (hunk 1 above)
Comment (CliffAgius) [reply docs/maui/behaviors/user-stopped-typing-behavior.md:12]: Yes I copied this from the XCT but now you mention it I agree I will think of some new words.

**Comment by PRMerger14** [comment]
## PRMerger Results
 | Issue | Description |
 --- | --- |
 Added File(s) | This PR contains added files. New files require human review.
 Yaml File(s) | This PR includes changes to .yml file(s) owned by another author.
 File Change Percent | This PR contains file(s) with more than 20% file change.
 Index File | This PR contains a change to the index file.

**Review by jfversluis — APPROVED** [review]

**Comment by CliffAgius** [comment]
Sorry @jfversluis I spotted that I forgot to update the Index with the changed description text so there is an update to the PR just for that part.

**Pull request merged**